src/utils/base.js

```diff
@@ -51,6 +51,15 @@ const convertFontSizeToPx = function (fontSize) {
   return convertedFontSize;
 };
 
+const hexToRGB = function (hex) {
```

Comment:
Instead of maintaining this piece of code ourselves it should probably use a helper. Polished has some excellent ones and we won't need to pull in the entire library ✨ 

https://polished.js.org/docs/#transparentize
  
or

https://polished.js.org/docs/#parsetorgb